release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,60 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Before this update, combined specification and status updates lists triggered unnecessary firmware upgrades, which caused system downtime. With this release, a firmware upgrade optimization skips unnecessary firmware upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-56766[OCPBUGS-56766])
+
+* Before this update, the `console-telemetry` plugin received a `Forbidden` error due to using the wrong API endpoint for tracking usage. As a consequence, the `Forbidden error in console-telemetry-plugin usage tracking` error occurred. With this release, the `console-telemetry` plugin posts usage data to `/api/metrics/usage` instead of `/metrics/usage`. As a result, the `console-telemetry`plugin does not receive a `Forbidden` error, which ensures accurate usage tracking. (link:https://issues.redhat.com/browse/OCPBUGS-58364[OCPBUGS-58364])
+
+* Before this update, the installer failed when {aws-first} credentials were not found and the survey was attempting to list all {aws-short} regions preventing users from creating the install-config. With this release, the installer no longer fails when {aws-short} credentials are not set, allow users to input them during the survey. (link:https://issues.redhat.com/browse/OCPBUGS-59155[OCPBUGS-59155])
+
+* Before this update, when a `HostedCluster` was configured with a proxy URL such as http://user:pass@host, the authentication header was not getting forwarded by the konnectivity proxy to the user proxy, failing authentication. With this release, the proper authentication header is sent when a user and password is specified in the proxy URL. (link:https://issues.redhat.com/browse/OCPBUGS-59503[OCPBUGS-59503])
+
+* Before this update, the `oc-mirror`did not detect Helm Chart images that used an aliased sub-chart. As a consequence, the Helm Chart images were missing after mirroring. With this release, the `oc-mirror`detects and mirrors Helm Chart images with an aliased sub-chart. (link:https://issues.redhat.com/browse/OCPBUGS-59798[OCPBUGS-59798])
+
+* Before this update, worker scale-up jobs that used Podman v5 with the `container-tools` module for `netavark` failed due to denial of the Open Container Initiative (OCI) permission when writing `devices.allow` with `crun`. As a consequence, the container scaling jobs failed. With this release, the `netavark` dependency for Podman is disabled, and `runc` runtime is used instead. As a result, worker scale-up jobs are now successful. (link:https://issues.redhat.com/browse/OCPBUGS-59843[OCPBUGS-59843])
```

Comment:
We don't use "worker" unless it is a code literal (i.e. `role = worker` in a config file), these should likely be "compute" instead